### ISSUE TITLE
Afficher l'historique d'une ressource

### ISSUE THREAD

**Issue by bouviermullerp** [assemblee-virtuelle/semapps#7]
**prérequis** : traçabilité

En plus de l'affichage des éléments qui le constitue. On a une idée de l'activité d'un noeud par les relations qui se sont créées, enlevées.

**Comment by srosset81** [comment]
Ce chantier va dépendre de celui-ci: https://github.com/assemblee-virtuelle/semapps/issues/448

**Comment by GuillaumeAV** [comment]
Exemple : Autour d'un thème j'aimerais avoir le fil d'actu du thème : Un projet a été créé en lien avec ce thème, un doc a été créé autour de ce thème, un événement est organisé autour de ce thème .. (Projet & evenement) sont des agents au sens de l'ontologie pair, mais ce n'est pas le cas de doc ou de thème ... Peuvent-ils avoir néanmoins des fils d'actus ?

**Comment by bouviermullerp** [comment]
Oui, je pense qu'ils peuvent. L'intention aussi est de faciliter le travail du modérateur et de savoir "qui modifie quoi". Si quelqu'un supprime un lien entre un doc et un thème, on doit pouvoir le tracer.

**Comment by srosset81** [comment]
Pour ref, il y a une issue dans le repo ActivityPub sur ce sujet:
https://github.com/w3c/activitypub/issues/343
